lib/geokit/geocoders/google.rb

```diff
@@ -179,8 +179,11 @@ def self.set_address_components(loc, addr)
             loc.district = comp['long_name']
           when types.include?('neighborhood')
             loc.neighborhood = comp['short_name']
+          # Use either sublocality or admin area level 3 if google does not return a city
           when types.include?('sublocality')
             loc.city = comp['long_name'] if loc.city.nil?
+          when types.include?("administrative_area_level_3")
+            loc.city = comp['long_name'] if loc.city.nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.